stable/scalyr/templates/cm-agent.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "configuration.fullname" . }}
```

Comment:
Is there a reason why you name the config map differently? I'd suggest to use `{{ template "fullname" . }}` as well.